frontend/src/components/pages/CampCreation/ScheduledSessionsRow.tsx

```diff
@@ -0,0 +1,72 @@
+import { Box, HStack, Text, Button } from "@chakra-ui/react";
+import React from "react";
+import { CreateCampSession } from "../../../types/CampsTypes";
+import SessionDayButton from "./SessionDayButton";
+
+type ScheduledSessionsRowProps = {
```

Comment:
Can we rename this to `Card` so it's more consistent to component naming in the project (it is a card technically right?)